src/main/asciidoc/reference/mapping.adoc

```diff
@@ -530,6 +533,8 @@ Being based on Spring's application context event infrastructure lets other prod
 
 To intercept an object before it goes into the database, you can register a subclass of `org.springframework.data.cassandra.core.mapping.event.AbstractCassandraEventListener` that overrides the `onBeforeSave(…)` method.
 When the event is dispatched, your listener is called and passed the domain object (which is a Java entity).
+Entity lifecycle events can be costly and you may notice a change in the performance profile when loading large result sets.
+You can disable lifecycle events on the Template API object.
```

Comment:
A link to the `setEnabled` methods in the API doc would be nice. At least we should name the method explicitly so the user doesn't have to search for the right method.